datalad/core/local/tests/test_save.py

```diff
@@ -216,14 +216,17 @@ def test_symlinked_relpath(path):
         ds.repo.add("mike1", git=True)
         ds.save(message="committing", path="./mike1")
 
-    # Let's also do in subdirectory
+    # Let's also do in subdirectory as CWD, check that relative path
+    # given to a plain command (not dataset method) are treated as
+    # relative to CWD
     with chpwd(op.join(dspath, 'd')):
-        ds.save(
-            message="committing", path=op.join(op.curdir, "mike2"))
+        save(dataset=ds.path,
+             message="committing",
+             path=op.join(op.curdir, "mike2"))
```

Comment:
Shouldn't `path` be just "mike2" here to prove the point?

Reply:
historically, yes. will change.

Reply:
Done